screens/onboarding/OnboardingScreen.js

```
@@ -0,0 +1,77 @@
+import { View, Text, TextInput, Button } from "react-native";
+import React, { useState } from "react";
+import Validation from "../../utils/Validation";
+import AsynncStorageManager from "../../local_storage/AsyncStorageManager";
+import StorageKeys from "../../local_storage/StorageKeys";
+import style from "./styles/OnboardingStyle";
+import textStyle from "../../design_token/TextStyle";
+import inputStyle from "../../design_token/InputStyle";
+import spacing from "../../design_token/Spacing";
+import String from "./res/String";
```

Biome (1.9.4)

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)